src/app/lists/page.tsx

```diff
@@ -1,19 +1,19 @@
 import { Metadata } from "next";
 import React, { ReactElement } from "react";
-import ListsDataView from "@/app/lists/ListDataview";
-import supabase, { Schema } from "@/supabase";
+import ListsDataView, { ListRow } from "@/app/lists/ListDataview";
+import supabase from "@/supabase";
 import Title from "@/components/Title/Title";
 
 // disables caching
 export const revalidate = 0;
 
-const fetchData = async (): Promise<Schema["lists"][]> => {
+const fetchData = async (): Promise<ListRow[]> => {
```

Comment:
How come this type was changed?

Reply:
I had to add `boolean` as a valid type that `Datum` columns can take to support the changes I made for conditional rendering of custom cells. After that, the type system had a lot of small complaints and it was a bit of a kafuffle to get it sorted

Long story short, it's just what I had to do to keep the type system happy